addons/account/i18n/de.po

```diff
@@ -808,7 +808,7 @@ msgstr ""
 #. module: account
 #: model_terms:ir.ui.view,arch_db:account.report_invoice_document
 msgid "<span>Unit Price</span>"
-msgstr "<span>Einzelpreis</span>"
+msgstr "<span>Stückpreis</span>"
```

Comment:
I don't see why not, but I don't see why it is said that it's not from transifex
![image](https://github.com/odoo/odoo/assets/93133576/510a8695-9b41-4488-a05e-ea8c51cf93ae)

Reply:
to be honest i didn't check transiflex, but when i tested it, the "unit price" wasn't correctly translated so don't know :rofl: